addons/mail_plugin/controllers/mail_plugin.py

```diff
@@ -291,3 +303,13 @@ def _mail_content_logging_models_whitelist(self):
         it can be overridden by sub modules in order to whitelist more models
         """
         return ['res.partner']
+
+    def _get_search_domain(self, email):
```

Comment:
_get_iap_search_term ? get_domain leads people to think this return an odoo domain. Or get_search_company_domain, email_domain ?

Reply:
`_get_iap_search_term` is nice :)